Packs/EmailCommunication/Scripts/DisplayEmailHtml/DisplayEmailHtml.py

```diff
@@ -0,0 +1,107 @@
+import json
+import re
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def create_email_html(email_html, entry_id_list):
```

Comment:
@altmannyarden change the function signature to:
`def create_email_html(email_html='', entry_id_list=[]):`